.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and deploy sphinx docs
+
+on:
+  push:
+    branches: main
+  pull_request:
```

Comment:
Maybe drop this trigger? Or should we keep it for some time? OK to keep it for a few days / weeks, though.

Reply:
we can remove it in the next PR. I was thinking that maybe we want to check the docs if anyone changes it, but I guess we can do that locally, as well.